r/src/embedding.r

```diff
@@ -39,6 +39,12 @@ runEmbedding <- function(req) {
                         min.dist = config$minimumDistance, 
                         metric = config$distanceMetric,
                         umap.method = "uwot-learn")                        
-        return(Embeddings(data, reduction = type))        
+        df_embedding <- Embeddings(data, reduction = type)
     }
+    
+    # Order embedding by cells id in ascending form
+    df_embedding <- df_embedding[rownames(data@meta.data[order(data@meta.data$cells_id), ]), ]
```

Comment:
Is this really necessary? As the cell ids are assigned to each cell depending on the barcode order when the object is created then they are already sorted by cellid

Reply:
I have added to remember that are ordered by cells id. Just in the case that any of the dimensional reduction techniques changes the order of the row.